go.mod

```diff
@@ -1,15 +1,27 @@
 module github.com/Khan/genqlient
 
-go 1.16
+go 1.18
 
 require (
 	github.com/99designs/gqlgen v0.17.2
-	github.com/agnivade/levenshtein v1.1.1 // indirect
 	github.com/alexflint/go-arg v1.4.2
 	github.com/bradleyjkemp/cupaloy/v2 v2.6.0
 	github.com/stretchr/testify v1.7.0
 	github.com/vektah/gqlparser/v2 v2.4.5
-	golang.org/x/sys v0.0.0-20220319134239-a9b59b0215f8 // indirect
 	golang.org/x/tools v0.1.10
 	gopkg.in/yaml.v2 v2.4.0
 )
+
+require (
+	github.com/agnivade/levenshtein v1.1.1 // indirect
+	github.com/alexflint/go-scalar v1.0.0 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/gorilla/websocket v1.4.2 // indirect
+	github.com/hashicorp/golang-lru v0.5.0 // indirect
+	github.com/mitchellh/mapstructure v1.2.3 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	golang.org/x/mod v0.6.0-dev.0.20220106191415-9b9b3d81d5e3 // indirect
+	golang.org/x/sys v0.0.0-20220319134239-a9b59b0215f8 // indirect
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
+)
```

Comment:
It seems like this is duplicated now? Did you edit this manually or are the go tools doing something weird?

Reply:
This is auto generated by `go tidy`. My editor was complaining it needed to be run, but I'm happy to remove ti.